src/internal/lokiutil/lokiutil.go

```diff
@@ -70,16 +65,10 @@ func QueryRange(ctx context.Context, c *loki.Client, queryStr string, from, thro
 			return nil
 		}
 		from = from.Add(time.Nanosecond)
-		// check if the context has been cancelled
```

Comment:
why remove this?

Reply:
We're can now pass context into `QueryRange` so i don't think it's needed anymore?

Reply:
I see, so if ctx is cancelled QueryRange would return an error